### PR TITLE
ZCS-2822: Changes to create distribution packages for timezones

### PR DESCRIPTION
**Pull request by shri314**

### CODE REVIEW

File: instructions/FOSS_staging_list.pl [hunk 1]
```diff
@@ -352,7 +350,7 @@
       "dir"         => "zm-zcs-lib",
       "ant_targets" => ["dist"],
       "stage_cmd"   => sub {
-         System("(cd .. && rsync -az --relative zm-zcs-lib $CFG{BUILD_DIR}/)");
+         System("(cd .. && rsync -az --relative --delete zm-zcs-lib $CFG{BUILD_DIR}/)");
```
Comment (fatkudu) [inline instructions/FOSS_staging_list.pl]: This seems unrelated and perhaps should have its own bug?

File: instructions/FOSS_staging_list.pl [hunk 2]
```diff
@@ -20,11 +20,9 @@
...
    },
    {
-      "dir"         => "zm-timezones",
-      "ant_targets" => ["publish-local"],
-      "stage_cmd"   => sub {
-         System("(cd .. && rsync -az --relative zm-timezones $CFG{BUILD_DIR}/)");
-      },
+      "dir"             => "zm-timezones",
+      "ant_targets"     => ["pkg"],
+      "deploy_pkg_into" => "zimbra-foss",
```
Comment (fatkudu) [inline instructions/FOSS_staging_list.pl]: I'm unsure what this means.  Will zm-timezones be in its own package - which would be useful for independant updating or will it be part of a zimbra-foss package which has lots of other baggage?

Comment (shri314) [inline instructions/FOSS_staging_list.pl]: Yes, with this commit, we are creating a different package out of zm-timezones, so that it can be potentially it be independently updated (out of the release cycle).

zimbra-foss is a logical name for the package repository that zimbra will have to host publically, so that users can be download them (via package management tools) from repository server. "deploy_pkg_into" is meant to specify a package repository name, where packages should get deployed when a build is created.

In my previous commit, I have disabled "deploy_pkg_into => zimbra-foss", that means the new package will simply get bundled into the tarball like others are, and we can continue our current tarball installations, until we are ready for repository deployments.

Comment (fatkudu) [inline instructions/FOSS_staging_list.pl]: Thanks for the explanation.  Much appreciated!
